### PR TITLE
Update Mojang Studios

### LINKED ISSUE #3087: Update Mojang Studios
**Name:** Mojang Studios
**Official resources for icon and color:**
image

https://www.minecraft.net/article/meet-mojang-studios
https://twitter.com/Minecraft/status/1262050538897670145
https://www.youtube.com/watch?v=YosWmbHAr2g

### PR DESCRIPTION
**Pull request by fbernhart**
mojangstudios

Closes #3087 
**Alexa rank:** [13,959](https://www.alexa.com/siteinfo/mojang.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
The stacked version of the new logo can be found here: https://www.minecraft.net/content/dam/franchise/logos/Mojang-Studios-Logo-Redbox.png

@EinsEli vectorized this into a SVG file in #3087. I took his work, removed the color and resized and centered the icon.

### CODE REVIEW

**Comment by adamrusted** [comment]
There is an SVG of this available on the [Minecraft Gamepedia Wiki](https://minecraft.gamepedia.com/File:Mojang_Studios_logo.svg) - how usable that is as a source I'll leave up to @simple-icons/maintainers

**Comment by ericcornelissen** [comment]
> There is an SVG of this available on the [Minecraft Gamepedia Wiki](https://minecraft.gamepedia.com/File:Mojang_Studios_logo.svg) - how usable that is as a source I'll leave up to other @simple-icons/maintainers

I'd say it can be used as a starting point, but not as an official source (not all _-pedia_ sites have standards for this kind of stuff, though we could consider _gamedia_ for the future if it turns out they generally contain good SVGs). If it matches the official PNG then that's great!

File: icons/mojangstudios.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...6169 0 00.0534-.6457 2.276 2.276 0 000-.2502c-.0022-.1484-.0015-.2984-.0022-.449 0-.4746.0498-.4307-.4388-.4315-.302 0-.302 0-.3035.2977 0 .3906.0066.7811-.0066 1.1702-.0088.2538-.1418.3795-.3561.3795-.2143 0-.3314-.1103-.3446-.3583-.0168-.3225-.0123-.6459-.0175-.9684-.0095-.6122.0731-.512-.5537-.5237zm-3.764.0008c-.2962 0-.2962 0-.2976.3056 0 .3555 0 .3554.3599.3562h.1572c.155.007.2143.0476.2113.2048-.008.3657 0 .7365 0 1.1044 0 .5748-.0446.485.512.501.1902.006.2523-.0688.2494-.2487-.0059-.4433 0-.8864 0-1.3296 0-.1704.0388-.2334.2319-.2318h.158c.343 0 .3437 0 .3415-.3533a.936.936 0 01-.0044-.1112c.0037-.1463-.0621-.2004-.2062-.1974-.3006.007-.6012 0-.901 0zm7.8202 0c-.174-.009-.2442.0548-.239.2324.0087.3306 0 .6583 0 .9918v1.0371c0 .136.0423.2122.1937.2085.308-.007.6173.011.9238-.0124.6582-.0497 1.1204-.4945 1.1585-1.081.0497-.7401-.2816-1.2068-1.0042-1.3355-.3357-.06-.6883-.0241-1.0328-.041zm.6104.6713c.5286-.0214.786.2899.6455.7657-.0732.256-.409.4447-.662.3657a.1463.1463 0 01-.0884-.103c-.0066-.3358-.0045-.6722-.0045-1.0196.0376-.004.074-.007.1093-.009zm7.0038.0121a.5464.5464 0 01.5497.5445.5259.5259 0 01-.5515.5515.5288.5288 0 01-.5413-.5397.5464.5464 0 01.543-.5563z"/></svg>
```
Comment (adamrusted) [inline icons/mojangstudios.svg:1]: <details>
<summary>There seem to be some differences between this (red) and the SVG source found (black).</summary>

</details>
<details>
<summary>Even with the PNG source (below), there are some small differences.</summary>

</details>
<details>
<summary>I've just pulled in the Gamepedia SVG, and it looks like the best possible match to the PNG file linked as source.</summary>

</details>

If there's no objection from any other @simple-icons/maintainers - I'd recommend that we use the **Gamepedia SVG** as our source - and can review at a later date, should the logo change and they can't keep up with changes.

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/mojangstudios.svg (hunk 1 above)
Comment (fbernhart) [reply icons/mojangstudios.svg]: @adamrusted Great work! I've now used the new SVG that you've provided.

Comment (adamrusted) [reply icons/mojangstudios.svg]: Fab - did you want to update the source to the SVG also?

Comment (fbernhart) [reply icons/mojangstudios.svg]: Quoting @ericcornelissen:

> I'd say it can be used as a starting point, but not as an official source

Comment (adamrusted) [reply icons/mojangstudios.svg]: Is the precision of 4 necessary on this?

Comment (fbernhart) [reply icons/mojangstudios.svg]: @adamrusted It's optimized with a precision of 3, isn't it? 🤔

Comment (adamrusted) [reply icons/mojangstudios.svg]: Yes. Don't mind me. Reading the preview from when I left the original comment 😅

**Review by adamrusted — APPROVED** [review]
LGTM! 🟩

**Pull request merged**